hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerProtocolServerSideTranslatorPB.java

```diff
@@ -79,7 +71,6 @@ public class OzoneManagerProtocolServerSideTranslatorPB implements OzoneManagerP
    * Only used to handle write requests when ratis is disabled.
    * When ratis is enabled, write requests are handled by the state machine.
    */
```

Comment:
Remove this comment
```suggestion
```

Reply:
Thanks for spotting it, the variable is also leftover.